test/util/edu-helpers.js

```diff
@@ -16,6 +18,35 @@ function completeVeteranInformation(client, data, onlyRequiredFields) {
   }
 }
 
+function completeRelativeInformation(client, data, onlyRequiredFields) {
+  const dobFields = data.relativeDateOfBirth.split('-');
+  client
+    .clearValue('input[name="root_relativeFullName_first"]')
+    .setValue('input[name="root_relativeFullName_first"]', data.relativeFullName.first)
```

Comment:
Unless we can bootstrap a little helper function onto `client` which just calls those two functions in order.

```js
function clearAndSetValue(selector, value) {
  this.clearValue(selector);
  return this.setValue(selector, value);
}
```
Or however it works out.

Just a thought.